search-books/src/main/java/nix/education/exception/InvalidFilterQueryException.java

```diff
@@ -0,0 +1,8 @@
+package nix.education.exception;
+
+public class InvalidFilterQueryException extends RuntimeException {
```

Comment:
I think we don't need to create a new exception class for this. IllegalArgumentException  should fit well your intention. Just provide the descriptive message for it

Reply:
Ok, I deleted this exception